packages/terra-tabs/src/common-tabs/_TabUtils.js

```diff
@@ -120,6 +120,12 @@ const lastFocus = (event, index, ids) => {
     }
   }
 };
+/**
+ * The default event handler for navigational arrow keys.
+ * @param {object} event The triggering event.
+ * @param {number} index The index of the current element.
+ * @param {array} ids The array of id strings.
+ */
```

Comment:
```suggestion
```
This looks like duplicate do we need this parameter details ?

Reply:
Done https://github.com/cerner/terra-framework/commit/628052153b88461ef88733be6b05c1fb519e3f41